Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -554,6 +596,7 @@ const u8* Jit64::DoJit(u32 em_address, PPCAnalyst::CodeBuffer *code_buf, JitBloc
 	jit->js.numFloatingPointInst = 0;
 
 	PPCAnalyst::CodeOp *ops = code_buf->codebuffer;
+	js.blockEnd = ops[code_block.m_num_instructions - 1].address;
```

Comment:
Deciding what is actually part of a block becomes a lot more complicated, but yes, essentially.